src/components/signup/index.jsx

```diff
@@ -0,0 +1,91 @@
+import styled from 'styled-components';
+import { Typography } from '@mui/material';
+import InputComp from '../inputfield/inputfield';
+import ButtonComponent from '../button/Button';
+import {
+  passwordIcon, userIcon, genderIcon, weightIcon, emailIcon, birthDataIcon, heightIcon,
+} from '../../utilize/icons';
+
+const SignupWrapper = styled.div`
+display: flex;
+flex-direction: row;
+jsutify-content: space-between;
+align-contetn: center;
+padding: 0rem;
+`;
+
+const FieldColumn = styled.div`
+display: flex;
+flex-direction: column;
+jsutify-content: space-between;
+align-contetn: center;
+padding: 2rem;
+gap:0.8rem;
+margin-bottom: rem; /* Add this line to increase the distance between columns */
+
+
+`;
+const CompWrapper = styled.div`
+width: 100%;
+max-width: 700px;
+margin:  auto;
+`;
+
+const ButtonContainer = styled.div`
+width: 70%;
+margin-left: 8.5rem;
+`;
+
+const Inputstyled = styled(InputComp)`
```

Comment:
here you are extends from a function component, not another style component, change this